### PR TITLE
update README_EN.MD

### PR DESCRIPTION
**Pull request by gamma63**
New revision of README_EN.MD (translated from README.MD)

### CODE REVIEW

File: README_EN.MD [hunk 1]
```diff
@@ -1,100 +1,272 @@
...
-<p>
-<img src="https://reporoster.com/forks/pimnik98/SayoriOS"/>
-</p>
-</p>
-</body></html>
+# SayoriOS - open source kernel
 
+![C](https://img.shields.io/badge/c-%2300599C.svg?style=for-the-badge&logo=c&logoColor=white) ![Workflow](https://github.com/pimnik98/SayoriOS/actions/workflows/SayoriOS-AutoBot-Auto.yml/badge.svg)
+
+Simple OS for PC's on x86 (i386) architecture. Writed in C, C++, ASM.
```
Comment (NDRAEY) [inline README_EN.MD:5]: ... Written in C and Assembly

File: README_EN.MD [hunk 2]
```diff
@@ -1,100 +1,272 @@
...
+    make run - Common run in QEMU with debug output to Qemu.log
+    make runlive - Common run in QEMU with debug output to console
+    make uefi - Run of emulator in UEFI mode
+    make geniso - Classic ISO build
+    make genuefi - Kernel build with UEFI
+    make clean - removes ISO file
+    make clean-objs - removes cache files
+    make link - link of the kernel files
+    make bir - Build of ISO and kernel run
+    make WSL - Buikd of ISO and kernel run from WSL
```
Comment (NDRAEY) [inline README_EN.MD]: ~~Buikd~~ Build

File: README_EN.MD [hunk 3]
```diff
@@ -1,100 +1,272 @@
...
+
+### <a id="run_qemu">Running in QEMU</a>
+
+Create a folder, move the kernel file to that folder, install software:
+
+    sudo apt-get install libvirt-daemon libvirt-clients bridge-utils virt-manager qemu-kvm qemu virt-manager
+
+cd to the folder and run emulator
+
+    qemu-system-i386 -cdrom kernel.iso -serial file:Qemu.log -accel kvm -m 128M -name "SayoriOS Soul" -d guest_errors -rtc base=localtime -soundhw pcspk
```
Comment (NDRAEY) [inline README_EN.MD:152]: Or also you can just do `make birl`

**Review by NDRAEY — CHANGES_REQUESTED** [review]
Some non-fatal errors were ignored, just feedback

File: README_EN.MD (hunk 2 above)
Comment (gamma63) [reply README_EN.MD]: one moment

File: README_EN.MD (hunk 1 above)
Comment (gamma63) [reply README_EN.MD:5]: In original README.MD there is no ASM badge

Comment (gamma63) [reply README_EN.MD:5]: if you meant the line 5 original file also mentioned the C++

**Pull request merged**